components/brave_wallet/browser/keyring_service.cc

```diff
@@ -2599,6 +2599,17 @@ KeyringService::SignCip30MessageByCardanoKeyring(
   return result;
 }
 
+std::array<uint8_t, kPolkadotSubstrateAccountIdSize>
+KeyringService::GetPolkadotPubKey(const mojom::AccountIdPtr& account_id) {
+  CHECK(account_id);
+
+  auto* keyring = GetKeyring<PolkadotKeyring>(account_id->keyring_id);
+  CHECK(keyring);
```

Comment:
Sorry, i was meaning that result should be optional here like in other similar places (KeyringService::GetBitcoinAddress for ex)

Reply:
Done! We now return `std::nullopt` when the `keyring` pointer is null,  because it'll be automatically null when the Polkadot feature is disabled.